doc/config.example.yaml

```diff
@@ -58,6 +58,20 @@ node:
   # increased in order to be added to the transaction pool
   double_spent_threshold_pct: 20
 
+  # true, if this node should collect statistics about other
+  # nodes in the network, including their geographical location and OS
+  collect_network_statistics : true
+
+  # The number of network crawlers that will be instantiated to collect
+  # statistics about other nodes in the network
+  num_of_crawlers : 3
+
+  # The number of seconds one crawler should wait after successfully contacted a node
+  crawling_interval_secs : 2
+
+  # The path to a file containing IP address -> geographical location mapping
+  ipdb_path : "data/ipdb.mmdb"
```

Comment:
I took a look at it locally, and I think `collect_network_statistics` and `ipdb_path` should be merged together.
As we do not want to give the user the chance to write a configuration that is self-inconsistent, we can base the fact that the user wants to use the frontend off the fact they provided an `ipdb_path` in the config file.
Also, `ipdb_path` => `mmdb_path`, as the later is more commonly used, and specific (`ipdb` would make sense if there was a standard for IP databases and there could be multiple providers).